Detectors/CTP/reconstruction/src/RawDataDecoder.cxx

```diff
@@ -273,13 +273,16 @@ int RawDataDecoder::decodeRaw(o2::framework::InputRecord& inputs, std::map<o2::I
       if (!mDoDigits) {
         continue;
       }
-      addCTPDigit(linkCRU, rdhOrbit, remnant, pldmask, digits);
+      addCTPDigit(linkCRU, rdhOrbit, remnant, pldmask, digitsMap);
       LOG(debug) << "diglet:" << remnant << " " << (remnant & bcmask).to_ullong();
       remnant = 0;
     }
   }
   if (mDoLumi) {
     lumiPointsHBF1.emplace_back(LumiInfo{orbit0, 0, 0, countsMBT, countsMBV});
+    for (auto const& dig : digitsMap) {
+      digits.push_back(dig.second);
+    }
```

Comment:
not sure about this: you are pushing digits w/o checking if they are requested?

Reply:
Fixed.